docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+* Aggregrations and Filtering 
+    * Grouping By Singular Variables
+    * Grouping By Multiple Variables 
+    * Filtering 
+* Using Known and Unknown Keys in Aggregrations
+* Data Preparation Limitations
+  * Limitations with ``with_columns``
+  * Limitations with ``filter`` 
+
+Dataset Description 
+-------------------
+
+The data is organized by year and quarter for each nation in the European Union. For this tutorial, we sampled 200,000 individuals from the public microdata of France across all study years. 
+
+The public microdata is protected using traditional statistical disclosure control methods such as global recoding, local suppression, and addition of noise. 
+The public microdata is protected using traditional statistical disclosure control methods such as global recoding, local suppression, and addition of noise. 
+
+
+Core Variables 
+--------------
+The `User Guide <https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-Database-UserGuide.pdf>`_ describes many variables. Our examples will use just a few. (Descriptions are copied from the User Guide.) 
+The `User Guide <https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-Database-UserGuide.pdf>`_ describes many variables. Our examples will use just a few. (Descriptions are copied from the User Guide.) 
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Variable
+     - Definition
+     - Coding
+   * - ``SEX``
+     - Sex
+     - 1: Male; 2: Female
+   * - ``AGE``
+     - Age of the Individual During the Reference Week
+     - Single Years
+   * - ``ILOSTAT``
+     - Labour Status During the Reference Week
+     - 1: Did any work for pay or profit during the reference week - one hour or more (including family workers but excluding conscripts on compulsory military or community service); 2: Was not working but had a job or business from which he/she was absent during the reference week (including family workers but excluding conscripts on compulsory military or community service); 3: Was not working because of lay-off; 4: Was a conscript on compulsory military or community service; 5: Other (15 years or more) who neither worked nor had a job or business during the reference week; 9: Not applicable (child less than 15 years old)
+   * - ``HWUSUAL``
+     - Number of Hours Per Week Usually Worked
+     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month; 01-98: Number of hours usually worked in the main job; 99: Not applicable; blank: No answer
+   * - ``QUARTER``
+     - Fixed Reference Quarter
+     - Single Quarter
+   * - ``YEAR``
+     - Fixed Reference Year
+     - Single Year
+
+Compositor Overview
+-------------------
+The compositor is the foundation of our differentially private queries. It essentially takes in our data and our specifications for the queries that we would like to run. At this point, we won't be directly referencing our data again and we could theoretically delete it! 
```

Comment:
It essentially takes .... and then does what with it? Maybe say a sentence about it.